src/makeRobot.js

```diff
@@ -39,6 +39,56 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    goForward(step = 1) {
+      if (step > 0) {
+        this.coords.y += step;
+      }
+
+      return this;
+    },
+    goBack(step = 1) {
```

Comment:
It's hard to read now. Let's improve the code readability by adding spaces where needed
```suggestion
    },
    
    goForward(step = 1) {
      if (step > 0) {
        this.coords.y += step;
      }

      return this;
    },
    
    goBack(step = 1) {
```